### PR TITLE
[T4A6][T13-B4] Aloisius Stephen

### PR DESCRIPTION
**Pull request by aloisiusStephen**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class UtilsTest {
+
+    private Utils utils;
+
+    @Before
```
Comment (z11i) [inline test/java/seedu/addressbook/common/UtilsTest.java:16]: Good job there using `@Before`.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    public void setup() {
+        utils = new Utils();
+    }
+    
+    @Test
+    public void nullInput_returnsTrue() {
```
Comment (z11i) [inline test/java/seedu/addressbook/common/UtilsTest.java:22]: Good naming.

**Comment by z11i** [comment]
Comments added. A very good job!

**Comment by aloisiusStephen** [comment]
thanks!

**Pull request closed without merging**